src/package.json

```diff
@@ -36,6 +35,12 @@
         "single-spa-angular/internals": "single-spa-angular-internals"
       }
     },
+    "whitelistedNonPeerDependencies": [
+      "single-spa"
+    ],
     "dest": "../lib"
+  },
+  "dependencies": {
+    "single-spa": "^5.5.1"
   }
```

Comment:
This should be removed.

Reply:
https://github.com/ng-packagr/ng-packagr/blob/master/docs/dependencies.md#why

Reply:
But the whole point of this PR is to get rid of the peerDependency, since we've had >10 people report the peer dependency as an issue.

Reply:
See related https://github.com/single-spa/single-spa-angular/issues/188

Reply:
Yeah, I understand the point of this PR. I'm just trying to consider all cases. The pasted link has a description of `package.dependencies` in _Why?_ section.

Wtyd?

Reply:
My motivation is to not have to constantly answer the same question over and over again -  there have been >10 people ask me about the single-spa dependency since we released single-spa-angular@4.

I think ng-packagr's reasoning for avoiding dependencies to be weak, and ultimately a concession that Angular's ecosystem isn't very well set up for dependency management.

To me, the ideal is to somehow get users to understand that they need to install single-spa as well as single-spa-angular. We could try to do this by updating the docs to do `ng add sinlge-spa-angular single-spa`. However, I'm not confident that users will read the docs. So the alternative is to mark it as a dependency with the best possible semver:

```
"single-spa": ">=5.3.0"
```

^ This still leaves room for duplicating single-spa dependency, which is bad. But at least there aren't lots of users reporting something that's not a bug as a bug.

Reply:
Then it might be changed to `>=5.3.0`?

Reply:
Apart from updating the package.json to expect `>=5.3.0`, should any other logic be changed? I couldn't figure out why those tests fail as they do, but I'm thinking that the current tests expect it to be a peerDependency, and something should be changed to reflect single-spa as a regular dependency.

Reply:
Sure, redirects is surely more critical. If you need help testing let me know, I'll keep investigating this issue from my end.